src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -440,11 +442,18 @@ const Authenticate = (() => {
             [`<a href="${Url.urlFor('contact')}">`, '</a>']);
         displayNotification(msg, false, 'document_under_review');
         setTimeout(() => {
-            $('#not_authenticated, #not_authenticated_financial').setVisibility(0); // Just hide it
+            removeButtonLoading();
+            $button.setVisibility(0);
```

Comment:
not sure if we want to hide the button, why not show disabled button? user might think they're not allowed to submit anymore